### PR TITLE
sort links alphabetically

### PR DESCRIPTION
**Pull request by bacongobbler**

### CODE REVIEW

File: docs/related.md [hunk 1]
```diff
@@ -6,46 +6,47 @@ add to this list, please open an [issue](https://github.com/kubernetes/helm/issu
...
+- [Honestbee's Helm Chart Conventions](https://gist.github.com/so0k/f927a4b60003cedd101a0911757c605a)
 - [Releasing backward-incompatible changes: Kubernetes, Jenkins, Prometheus Operator, Helm and Traefik](https://medium.com/@enxebre/releasing-backward-incompatible-changes-kubernetes-jenkins-plugin-prometheus-operator-helm-self-6263ca61a1b1#.e0c7elxhq)
-- [CI/CD with Kubernetes, Helm & Wercker ](http://www.slideshare.net/Diacode/cicd-with-kubernetes-helm-wercker-madscalability)
-- [The missing CI/CD Kubernetes component: Helm package manager](https://hackernoon.com/the-missing-ci-cd-kubernetes-component-helm-package-manager-1fe002aac680#.691sk2zhu)
 - [The Workflow "Umbrella" Helm Chart](https://deis.com/blog/2017/workflow-chart-assembly)
-- [GitLab, Consumer Driven Contracts, Helm and Kubernetes](https://medium.com/@enxebre/gitlab-consumer-driven-contracts-helm-and-kubernetes-b7235a60a1cb#.xwp1y4tgi)
+- [The missing CI/CD Kubernetes component: Helm package manager](https://hackernoon.com/the-missing-ci-cd-kubernetes-component-helm-package-manager-1fe002aac680#.691sk2zhu)
```
Comment (hickeyma) [inline docs/related.md]: Should this be above "The Workflow "Umbrella" Helm Chart]"?

Comment (bacongobbler) [reply docs/related.md]: vim sorts capitalized words as a higher precedence over lowercased words. I'll fix the title so it's consistent.

Comment (hickeyma) [reply docs/related.md]: Ok, thanks.

**Review by hickeyma — APPROVED** [review]

**Comment by hickeyma** [comment]
LGTM

**Review by SlickNik — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**